games/devtest/mods/testnodes/textures.lua

```diff
@@ -129,17 +142,21 @@ for i=1, #fractal do
 		b = floor(abs(1 - fractal[i]) * 255),
 		a = 255,
 	}
-	data_ck[i] = checker[i] > 0 and "#F80" or "#000"
+	data_ck[i] = checker[i] > 0 and "#888" or "#000"
 end
 
+fractal = nil
+frac_emb = nil
+checker = nil
```

Comment:
You could put a `do` + `end` block around the relevant code to automatically let the variables run out of scope. Diff viewers can ignore whitespace changes.

Reply:
it's a bunch of lines so meh